File: XOutput.Api/Websocket/Input/InputDeviceFeedbackResponse.cs

using System.Collections.Generic;

namespace XOutput.Websocket.Input
{
    public class InputDeviceFeedbackResponse : MessageBase
    {
        public const string MessageType = "InputDeviceFeedback";
        public List<InputDeviceTargetValue> Targets { get; set; }
        public InputDeviceFeedbackResponse()
        {
            Type = MessageType;
        }
    }
}

File: @xoutput/api/src/websocket/input/InputDeviceFeedbackResponse.ts
import { MessageBase } from '../MessageBase';
import { InputDeviceTargetValue } from './InputDeviceTargetValue';

export const InputDeviceFeedbackResponseType = 'InputDeviceFeedback';

export type InputDeviceFeedbackResponse = MessageBase<typeof InputDeviceFeedbackResponseType> & {
  targets: InputDeviceTargetValue[];
};
